components/x11/libXfont/mapfile-vers

```diff
@@ -183,13 +183,9 @@ SUNWprivate {
 	pcfWriteFont;
 	pcfError;
 
-	# "src/bitmap/snfstr.h" - needed by xfs
-	SnfSetFormat;
```

Comment:
No no it is not that bad, only xfs, installalias and bdftopcf are affected.
I just saw you merged the other branches, I thought we were waiting a bit for feedback.
I can remove it for now.